src/cardsort/analysis.py

```diff
@@ -13,6 +14,9 @@
 ]
 
 
+logging.basicConfig(level=logging.INFO)
```

Comment:
This should not be configured here. Ultimately, the end user should be the one to decide what to do with the logs (should I print them in the console? ignore them? how should I format them?).

Use this in `__init__.py` :
```py
import logging
logging.getLogger(__name__).addHandler(logging.NullHandler())
```
Here, use this instead
```py
logger = logging.getLogger(__name__)
```

Then, you may use `logger.info()` and other methods in this file.

See this part of the doc for the complete rationale:  https://docs.python.org/3/howto/logging.html#configuring-logging-for-a-library

Reply:
Thank you @Batalex, that makes sense. I fixed it in [3f1b18e](https://github.com/katoss/cardsort/pull/5/commits/3f1b18e90ca20d53478b2be1b7c8bc14c3681638) :)

Reply:
You forgot to change the instances of `logging.method` to `logger.method`

Reply:
Thank you, fixed it in [3867cfc](https://github.com/katoss/cardsort/pull/5/commits/3867cfceed3b65c0eb8d745bb6357a69d9c27443)